README.md

```diff
@@ -91,61 +91,247 @@ git remote add upstream-code https://github.com/microsoft/vscode
+
+$ docker build -f build/dockerfiles/linux-libc.Dockerfile -t linux-libc-amd64 .
+
+$ export DOCKER_BUILDKIT=1
+
+$ docker build -f build/dockerfiles/assembly.Dockerfile -t vs-code-open-source:next .
+
+$ docker push <username>/vs-code-open-source:next
+```
+
+8\. Create a `/.che/che-editor.yaml` file in the remote repository that you intend to clone into workspaces. This file must specify the container image of your customized Visual Studio Code - Open Source that is to be pulled for new workspaces.
+
+*Example. `/che-editor.yaml` for the branded Visual Studio Code - Open Source*
+
+```yaml
+inline:
+  schemaVersion: 2.1.0
+  metadata:
+    name: che-code
+  commands:
+    - id: init-container-command
+      apply:
+        component: che-code-injector
+  events:
+    preStart:
+      - init-container-command
+  components:
+    - name: che-code-runtime-description
+      container:
+        image: quay.io/devfile/universal-developer-image:ubi8-latest
+        command:
+          - /checode/entrypoint-volume.sh
+        volumeMounts:
+          - name: checode
+            path: /checode
+        memoryLimit: 2Gi
+        memoryRequest: 256Mi
+        cpuLimit: 500m
+        cpuRequest: 30m
+        endpoints:
+          - name: che-code
+            attributes:
+              type: main
+              cookiesAuthEnabled: true
+              discoverable: false
+              urlRewriteSupported: true
+            targetPort: 3100
+            exposure: public
+            secure: false
+            protocol: https
+          - name: code-redirect-1
+            attributes:
+              discoverable: false
+              urlRewriteSupported: true
+            targetPort: 13131
+            exposure: public
+            protocol: http
+          - name: code-redirect-2
+            attributes:
+              discoverable: false
+              urlRewriteSupported: true
+            targetPort: 13132
+            exposure: public
+            protocol: http
+          - name: code-redirect-3
+            attributes:
+              discoverable: false
+              urlRewriteSupported: true
+            targetPort: 13133
+            exposure: public
+            protocol: http
+      attributes:
+        app.kubernetes.io/component: che-code-runtime
+        app.kubernetes.io/part-of: che-code.eclipse.org
+    - name: checode
+      volume: {}
+    - name: che-code-injector
+      container:
+        image: quay.io/username/vs-code-open-source:next
+        command: ["/entrypoint-init-container.sh"]
+        volumeMounts:
+          - name: checode
+            path: /checode
+        memoryLimit: 128Mi
+        memoryRequest: 32Mi
+        cpuLimit: 500m
+        cpuRequest: 30m
+```
+
+:grey_exclamation: In this example, `quay.io/username/vs-code-open-source:next` specifies the container image of a branded Visual Studio Code - Open Source that will be pulled at workspace creation.
```

Comment:
since the likely outcome is an organization path, not a username path, should we suggest a more likely example?

```suggestion
:grey_exclamation: In this example, `quay.io/branding-organization/vs-code-open-source:next` specifies the container image of a branded Visual Studio Code - Open Source IDE that will be pulled at workspace creation.
```

Reply:
Good catch.